examples/stubbing-spying__route2/cypress/integration/spy-on-fetch-spec.js

```diff
@@ -0,0 +1,18 @@
+/// <reference types="Cypress" />
+
+describe('route2', () => {
+  context('spying', function () {
+    beforeEach(function () {
+      cy.route2('/favorite-fruits').as('fetchFruits')
+      cy.visit('/')
+    })
+
+    it('requests favorite fruits', function () {
+      cy.wait('@fetchFruits')
+      // TODO: can we inspect the response object from the server?
+      // like response body...
```

Comment:
check out the object yielded from `cy.wait`, it will have this signature: https://github.com/cypress-io/cypress/blob/fe4e11ec6eba717dd8ed282394f5b968a31860a2/packages/net-stubbing/lib/external-types.ts#L76-L99

Reply:
Nope, it is missing

<img width="910" alt="Screen Shot 2020-09-09 at 12 17 53 PM" src="https://user-images.githubusercontent.com/2212006/92625364-8a728a80-f296-11ea-8eaa-c132a6a59e79.png">

Reply:
ah yes, this is probably the result of a optimization where, if the response is not intercepted on the driver, it is not sent. but we still need it in the case of a .wait like this...

Reply:
it's difficult because there is a decent performance hit to serialize the body, send it to the driver, deserialize it, so it sucks to do that for every request/response...

Reply:
well, created an issue: https://github.com/cypress-io/cypress/issues/8536